src/electron-main.js

```diff
@@ -814,6 +814,8 @@ protocol.registerSchemesAsPrivileged([{
 // transition into the user's browser.
 app.enableSandbox();
 
+app.commandLine.appendSwitch('disable-features', 'HardwareMediaKeyHandling,MediaSessionService');
```

Comment:
```suggestion
// We disable media controls here. We do this because calls use audio and video elements and they sometimes capture the media keys. See https://github.com/vector-im/element-web/issues/15704
app.commandLine.appendSwitch('disable-features', 'HardwareMediaKeyHandling,MediaSessionService');
```

Because this seems to be the only way, let's at least add a comment so that we know why are we doing this